flowmachine/flowmachine/core/server/query_schemas/spatial_aggregate.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marshmallow import Schema, fields, post_load
+from marshmallow_oneofschema import OneOfSchema
+
+
+from flowmachine.features import Flows
+from flowmachine.features.utilities.spatial_aggregates import SpatialAggregate
+from .base_exposed_query import BaseExposedQuery
+from .daily_location import DailyLocationSchema, DailyLocationExposed
+from .modal_location import ModalLocationSchema, ModalLocationExposed
+from .custom_fields import AggregationUnit
+
+__all__ = ["SpatialAggregateSchema", "SpatialAggregateExposed"]
+
+
+class InputToSpatialAggregate(OneOfSchema):
```

Comment:
At some point we should perhaps come up with more meaningful names for these "input" schemas. But the only alternative I can think of off the top of my head is `SpatiallyAggregatableQuery`, which isn't really much better. 😄